common/src/main/java/com/wynntils/screens/partymanagement/PartyManagementScreen.java

```diff
@@ -126,174 +150,145 @@ public void render(PoseStack poseStack, int mouseX, int mouseY, float partialTic
 
         super.render(poseStack, mouseX, mouseY, partialTick);
 
+        // region debug dividers
+        // to make positioning easier, we will split the screen into GRID_DIVISIONS parts horizontally and vertically
+        // uncomment when changing gui elements
```

Comment:
This section contains the commented debug grid I created for positioning elements. I think leaving it here would be helpful for anyone that tries to move things around in the future, but if anyone thinks this is too messy, feel free to lmk and I'll remove it

Reply:
We never left any code like this in main, I think that is how we should do it. Maybe we can introduce a `stash` folder in the repo, to have debug/whatever code like this. (I know we have DebugFeatures but those are vastly different)

Reply:
`stash` folder or `stash` branch? I don't think either is a super good idea.

I think it can be reasonable to save debug code, in place, like here. In fact, I'm not sure you even need to comment it out; if we have a method that is never called, but a comment saying "don't remove, used when debugging" I think it's probably better.